src/AdminConsole/Pages/App/Playground/NewAccount.cshtml

```diff
@@ -47,6 +47,40 @@
                                     <span asp-validation-for="Nickname"></span>
                                 </div>
                             </div>
+                            
+                            <div>
+                                <label class="block text-sm font-medium leading-6 text-gray-900">Attestation</label>
+                                <div class="mt-2 flex">
+                                    <div class="flex-1 radio-control">
+                                        <input asp-for="Attestation" type="radio" name="attestation" id="attestation-none" class="radio-input" value="none"/>
+                                        <label for="attestation-none" class="radio-label">None</label>
+                                    </div>
+                                    @if (CurrentContext.Features.AllowAttestation)
+                                    {
+                                        <div class="flex-1 radio-control">
+                                            <input asp-for="Attestation" type="radio" name="attestation" id="attestation-direct" class="radio-input" value="direct" />
+                                            <label for="attestation-direct" class="radio-label">Direct</label>
+                                        </div>
+                                        <div class="flex-1 radio-control">
+                                            <input asp-for="Attestation" type="radio" name="attestation" id="attestation-indirect" class="radio-input" value="indirect"/>
+                                            <label for="attestation-indirect" class="radio-label">Indirect</label>
+                                        </div>
+                                    }
+                                    else
+                                    {
+                                        <div class="flex-1 radio-control">
+                                            <input type="radio" name="attestation" id="attestation-direct" class="radio-input" disabled="disabled" />
+                                            <label for="attestation-direct" class="radio-label">Direct</label>
+                                        </div>
+                                        <div class="flex-1 radio-control">
+                                            <input type="radio" name="attestation" id="attestation-indirect" class="radio-input" disabled="disabled" />
+                                            <label for="attestation-indirect" class="radio-label">Indirect</label>
+                                        </div>
```

Comment:
Adda a small info/disclaimer that attestation is only available for enterprise plans, link to documentation what Attestation is?